### PR TITLE
create_deckkeyword

### PR DESCRIPTION
**Pull request by stefoss23**
This is a small part of issue #990.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: python/cxx/parser.cpp [hunk 1]
```diff
@@ -60,7 +62,8 @@ void opmcommon_python::export_Parser(py::module& module) {
 
 
     py::class_<ParserKeyword>(module, "ParserKeyword")
-        .def_property_readonly("name", &ParserKeyword::getName);
+        .def_property_readonly("name", &ParserKeyword::getName)
+        .def("create_deckkeyword", &create_deckkeyword, ref_internal);
```
Comment (joakim-hove) [inline python/cxx/parser.cpp]: Is there a reason you did not bind this to the `DeckKeyword` constructor the normal way?

Comment (stefoss23) [reply python/cxx/parser.cpp]: DeckKeyword has an internal pointer to a ParserKeyword, which need to be kept alive.

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
OK - this was just what the doctor ordered :+1:  - but the world has changed; and the doctor has a new wish .... The `DeckKeyword` constructor now *must* have a valid `ParserKeyword` argument - i.e. we can change the constructor:
```
DeckKeyword(const ParserKeyword& pkw);
```
and the member variable can be `ParserKeyword`value instance. Cool if you update the PR to take this account. Sorry about the back-and-forth.

**Comment by stefoss23** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this opm-upscaling=276 please

**Comment by bska** [comment]
jenkins build this opm-upscaling=276 please

**Comment by stefoss23** [comment]
jenkins build this opm-upscaling=276 please

**Comment by joakim-hove** [comment]
This has been merged as #1042

**Pull request closed without merging**